src/app/wishpool/(viewer)/final/layout.tsx

```diff
@@ -0,0 +1,26 @@
+'use client';
+import { useRouter } from 'next/navigation';
+
+import BackHeader from '@/components/layout/Header/BackHeader';
+import { PATH } from '@/constants/common/path';
+
+const FinalLayout = ({ children }: { children: React.ReactNode }) => {
+  const router = useRouter();
+
+  return (
+    <>
+      <BackHeader
+        title="선택 선물 확인하기"
+        bgColor="white"
```

Comment:
아마 헤더도 요 색인듯 합니닷 ..?!

```suggestion
        bgColor="background-02"
```